src/explore-education-statistics-admin/src/pages/release/data/ReleaseDataFilePage.tsx

```diff
@@ -52,9 +52,25 @@ const ReleaseDataFilePage = ({
   const getReplacementPlanMessage = () => {
     if (replacementDataFileError) {
       return (
-        <WarningMessage>
-          There was a problem loading the data replacement information.
-        </WarningMessage>
+        <>
```

Comment:
We're declaring the same cancel button twice. Let's refactor this entire function to avoid duplication. I would suggest something like this:

```tsx
const getReplacementPlanMessage = () => {
  if (replacementDataFile?.status === 'COMPLETE') {
    return null;
  }

  const cancelButton = (
    <>
      <Button>Cancel data replacement</Button>
      <ModalConfirm>...</ModalConfirm>
    </>
  );
  
  // the rest of the code
}
```